docs/source/analysis.ipynb

```diff
@@ -0,0 +1,530 @@
+    "!jupyter nbextension enable --py --sys-prefix widgetsnbextension\n",
+    "!jupyter nbextension enable --py --sys-prefix  --py qgrid\n",
+    "\n",
+    "# Install bokeh extensions\n",
+    "!jupyter nbextension install --sys-prefix --symlink --py jupyter_bokeh\n",
+    "!jupyter nbextension enable jupyter_bokeh --py --sys-prefix"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Progress\n",
+    "\n",
+    "**[IMPORTANT] ALWAYS run this cell when you open this notebook.** If you don't, the analysis functions won't work. Please ensure you run the Load Progress cell below before running any analysis modules.\n",
+    "\n",
+    "MoSeq uses a [progress.yaml](https://github.com/dattalab/moseq2-app/wiki/Directory-Structures-and-yaml-Files-in-MoSeq-Pipeline#progressyaml) file to keep track of the progress. \n",
+    "\n",
+    "The following cell generates a `progress.yaml` file or loads your progress from the last saved checkpoint.  This cell checks if there is a  `progress.yaml` in the project folder and generate/load a dictionary contains the necessary paths to use the features in the rest of the notebook. The cell output displays the content in `progress_paths`.\n",
+    "\n",
+    "- **Specify the filename to the progress file**. No change is needed if the notebooks are in the project folder.\n",
+    "- **Run the following cell**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keypoint_moseq.analysis import track_progress\n",
+    "\n",
+    "project_dir = './' # project directory\n",
+    "model_dirname = 'model_dir' # model directory name for the model to analyze\n",
+    "input_dir = './videos' # input video directory path\n",
+    "progress_filename = 'progress.yaml' # progress file name\n",
+    "\n",
+    "progress_paths = track_progress(model_dirname, project_dir, input_dir, 'progress.yaml')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Select a different model to analyze"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from os.path import join\n",
+    "from keypoint_moseq.analysis import update_model_progress\n",
+    "\n",
+    "model_dirname = model_dirname # select desired model name\n",
+    "\n",
+    "# update all relevant paths for the new model\n",
+    "progress_paths = update_model_progress(progress_paths, model_dirname, join(project_dir, 'progress.yaml'))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Assign Groups\n",
+    "\n",
+    "Sessions can be given group labels for analyses comparing different cohorts or experimental conditions and the labels will be stored in the `index.yaml` file. You can find more abaout this widget in the wiki.\n",
```

Comment:
@calebweinreb What wiki does this refer to? Does it exist yet? can we hyperlink? 

I removed all reference to the wiki for now: 42af5ac0c1d769ade75bc2b5061d026f2812e40d